kotlin-react-router/build.gradle.kts

```diff
@@ -7,6 +7,7 @@ plugins {
 dependencies {
     api(project(":kotlin-browser"))
     api(project(":kotlin-remix-run-router"))
+    api(project(":kotlin-react"))
```

Comment:
`kotlin-react-core`?

Reply:
`kotlin-react-core` is syntetic module just for `Space` usage, I don't want to be dependent of it, because it can be removed in futere.

Reply:
`kotlin-react-core` is what you need :)

Reply:
What do you mean?

Reply:
I propose to discuss it separately.